dashboard/test/models/level_group_test.rb

```diff
@@ -391,19 +391,19 @@ def get_external_dsl(id)
     level_group.stubs(:dsl_text).returns(level_group_input_dsl)
 
     # Copy the level group and all its sub levels.
-    level_group_copy1 = level_group.clone_with_suffix(' copy1')
```

Comment:
This is making me wonder what happens if there is space at the beginning of the suffix?

Reply:
And do we want to allow suffix to start with a space?

Reply:
Hmm good questions. My gut says that we should allow it as `name_suffix` should handle a lot of this for us. But, it doesn't look like that's working correctly for DSL levels (it's not set on any DSL levels).

This change was specifically to ensure that we have more readable names when we clone, so we don't end up with a level name like 'U10 SSQRP1 part 12022' again. I can definitely `strip` the suffix though if we think that would clearer? I'm just not sure doing a lot of transformations on the suffix is clearer

Reply:
Ya I think you are right its not a big deal to have a space in the level names it would only be the url that would be an issue.